dtests/src/test/java/io/snappydata/hydra/local.smartConnectorMode.conf

```diff
@@ -0,0 +1,14 @@
+hydra.HostPrms-hostNames =
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${snappyStoreHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> \", ${leadHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${locatorHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${workerHosts})" ncf
+;
+
+io.snappydata.hydra.cluster.SnappyPrms-serverLauncherProps = " -heap-size=4g -memory-size=8g -critical-heap-percentage=95 -member-timeout=60000 -J-Dp2p.socket_timeout=60000 ";
```

Comment:
Remove -critical-heap-percentage=95 as this is specific to ct bt run. It will cause problem while running eviction tests in northWind bt.